tests/scalability/README.md

```diff
@@ -94,3 +94,45 @@ From here we get the ddocs update sequences. Stage the branch which is associate
 Once the medic script completes we begin setting up for jmeter. The `start_jmeter_ec2.sh` script launches a new ec2 instance with the `user-data` script at `cht-core/tests/scalability/run_suite.sh`. 
 
 The script clones cht-core and navigates to the scalability dir. Then installs java, nodejs, jmeter, and its plugins. Executes the scalability suite which is defined in the `cht-core/tests/scalability/sync.jmx`, and finally it uploads the results to an s3 bucket. 
+
+
+
+## Distributed Workflow
+
+![Diagram of our current setup.](jmeter_distributed.png)
+
+Our current setup has 1 ec2 instance acting as the JMeter Orchestrator with 2 worker nodes. This can generate 1400 virtual users. Against our Application environment under test. 
+
+### Setup
+
+1. Launch an ec2 instance c5.2xlarge(Orchestrator)
+1. Launch 2 ec2 instances of c5.4xlarge(Worker)
+1. SSH into each of these machines. 
+1. Copy `tests/scalability/run_suite.sh` to the user directory. From this branch.
+1. On each of the three VMs, call  `chmod +x run_suite.sh&&./run_suite.sh` . This clones cht-core, installs JAVA, installs Node 14, runs npm install, installs JMeter, and its plugins.
+1. On just the orchestrator node, update the `cht-core/tests/scalability/config.json` to ensure the users and URL are correct.
+1. Split the `config.json` on the orchestrator
+    1. `cd cht-core/tests/scalability`
+    1. `npm install` 
+    1. Using `node tests/scalability/ec2/split_config.js --workerCount 2 --filePath = './config.json`
+    
+    NOTE: JMeter does not send any data to the workers. All it does is trigger the action to run as if you were doing it manually on that machine. You will need to split the `config.json` to have users for each worker node. 
+1. On the orchestrator node, create a [RMI key](https://jmeter.apache.org/usermanual/remote-test.html#setup_ssl) 
+1. Copy `rmi_keystore.jks` (RMI Key) and `config.json` split to the two worker nodes into `cht-core/tests/scalability/.`. Each worker node should get a different config file.
```

Comment:
```suggestion
1. Copy `rmi_keystore.jks` (RMI Key) and `config-N.json` files (one per worker node, `config-0.json`  and `config-1.json`) to the two worker nodes into `cht-core/tests/scalability/.`. Each worker node should get a different config file.
```

Reply:
Added a bit more detail on what happens when split and ensuring the generated file `conf-N` is copied and renamed to `config.json`